cmd/ddev/cmd/auth-ssh.go

```diff
@@ -81,7 +106,7 @@ var AuthSSHCommand = &cobra.Command{
 }
 
 func init() {
-	AuthSSHCommand.Flags().StringVarP(&sshKeyPath, "ssh-key-path", "d", "", "full path to SSH key directory")
+	AuthSSHCommand.Flags().StringVarP(&sshKeyPath, "ssh-key-path", "d", "", "full path to SSH key directory or file")
```

Comment:
It's awkward that -d would be for a file. I guess we could at least add aliases for file?